apps/prairielearn/src/lib/question-render.sql

```diff
@@ -166,6 +166,40 @@ WITH
       gj.id DESC
     LIMIT
       1
+  ),
+  -- The following two CTEs are copied from `selectAndAuthzInstanceQuestion`.
```

Comment:
This was the quickest way to fix things. I could also move this to a separate query that we could use in both the middleware and here... 🤷

Reply:
Copying for seems completely reasonable for now -- we can lift this out into proper library or model functions later.